posthog/session_recordings/session_recording_api.py

```diff
@@ -823,6 +851,76 @@ def _send_realtime_snapshots_to_client(
         else:
             raise exceptions.ValidationError(f"Invalid version: {version}")
 
+    @extend_schema(description="Generate session recording filters using AI")
+    @action(methods=["POST"], detail=False, url_path="ai/filters")
+    def ai_filters(self, request: Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
we could pull these new APIs out into their own file... since this is getting massive... but again not blocking since we're going to keep working here